spec.html

```diff
@@ -19962,7 +19962,7 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
               <pre><code class="javascript">constructor() {}</code></pre>
               using the syntactic grammar with the goal symbol |MethodDefinition[~Yield, ~Await]|.
         1. Set the running execution context's LexicalEnvironment to _classScope_.
-        1. Let _constructorInfo_ be DefineMethod of _constructor_ with arguments _proto_ and _constructorParent_.
+        1. Let _constructorInfo_ be ! DefineMethod of _constructor_ with arguments _proto_ and _constructorParent_.
```

Comment:
Using the ?/! feels awkward with this form - meaning where it doesn't look like a JS function invocation.

Reply:
Well, ...
- The spec already has 45 occurrences of this form.
- It looks like you yourself were the first to add it: 9405a43e (lines 7060 and 7062)
- If PR #955 goes through, such invocations could be changed to a more javascript-y form anyway.

Reply:
True enough :-) it still looks weird to me.